src/main/java/org/folio/consortia/domain/repository/UserTenantRepository.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.consortia.domain.repository;
+
+import org.folio.consortia.domain.entity.UserTenantEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UUID> {
+  Optional<UserTenantEntity> findByUserId(UUID id);
```

Comment:
We can have multiple tenants associated with one user id, so this method should return List

Reply:
we are getting user data because we have @ManyToOne relationship with Tenant. I think One object is enough in this case